pkg/ddb/expr_filter.go

```diff
@@ -28,3 +30,32 @@ func (b *filterBuilder) buildFilterCondition(metadata *Metadata) *expression.Con
 
 	return &expr
 }
+
+type ttlStruct struct {
```

Comment:
Can we put these two type definitions on top of the file?  
I always find it confusing, if there is something between functions for another receiver..